tfe/resource_tfe_workspace.go

```diff
@@ -200,7 +200,11 @@ func resourceTFEWorkspaceRead(d *schema.ResourceData, meta interface{}) error {
 	id := d.Id()
 	log.Printf("[DEBUG] Read configuration of workspace: %s", id)
 	workspace, err := tfeClient.Workspaces.ReadByID(ctx, id)
-	if err != nil && err != tfe.ErrResourceNotFound {
+	if err != nil {
+		if err == tfe.ErrResourceNotFound {
```

Comment:
might be worth adding a debug log here like the other resources do when they handle similar errors
```suggestion
		if err == tfe.ErrResourceNotFound {
		log.Printf("[DEBUG] Workspace %s no longer exists", d.Id())

```

Reply:
👍